test/controllers/users/omniauth_callbacks_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class Users::OmniauthCallbacksControllerTest < ActionController::TestCase
+
+  test "github signup" do
+    OmniAuth.config.add_mock(:github, {:uid => '12345'})
```

Comment:
Right now we have only one external API(Github) for signup/sign-in needs mocking. Github request mocking is provided by omniauth-github gem. So at this moment, there is no need of request mocking framework.